src/typings/index.d.ts

```diff
@@ -0,0 +1,35 @@
+type PokemonCardObject = {
+  id: string
+  name: string
+  supertype: string
+  types: string[]
+  images: {
+    small: string
+    large: string
+  }
+  set: {
+    releaseDate: string
+  }
+  flavorText: string
+  tcgplayer?: {
+    url: string
+  }
+}
+
+type ResponseDataType = {
```

Comment:
```suggestion
type ListCardsResponse = {
```

I'd be more explicit with this type name, especially since it's going in the global namespace.